modules/ROOT/pages/producing-ign.adoc

```diff
@@ -0,0 +1,64 @@
+:experimental:
+= Fedora CoreOS - Producing an Ignition File
+
+== Scope
+This guide provides easy, step-by-step instructions to produce a basic Ignition file.
+
+Fedora CoreOS (FCOS) Ignition files specify the configuration of FCOS. The process begins with a YAML configuration file. The FCOS Configuration Transpiler (FCCT) converts the human-readable YAML file into machine-readable JSON.
+
+TIP: Once FCOS boots and ingests the Ignition file, the configuration is immutable. Therefore, plan your Ignition file with the configuration that you will need.
+
+== A Simple Example
+This section provides instructions to create a basic Ignition file that produces a user named `fcos_user` that can SSH in with a key.
+
+The overall steps are as follows:
+
+. Write the Fedora CoreOS Configuration (FCC) file in the YAML format.
+. Download and install the Fedora CoreOS Configuration Transpiler (fcct).
+. Use fcct to convert the human-readable YAML FCC file into a machine-readable JSON Ignition file.
+
+=== Prerequisite
+Create the `fcos_user` on your local machine and generate an SSH key for this user. When you SSH into FCOS, you will use the SSH key from the local host.
+
+=== Writing the FCC File
+Copy the following example into a text editor:
+
+[source,yaml]
+----
+variant: fcos
+version: 1.0.0
+passwd:
+  users:                  #this section lists all users
+    - name: fcos_user     #the first username goes here
+      ssh_authorized_keys:
+        - ssh-rsa AAAA... #replace the line starting from "ssh" with the contents of the fcos_user's ssh public key file.
+----
+
+Save the file with the name `example.yaml`.
+
+TIP: YAML files must have consistent indentation. Although `fcct` checks for syntax errors, ensure that the indentation matches the above example. Overall, the FCC files must conform to ``fcct``'s schema.
+
+=== Downloading and Installing FCCT
+`fcct`, the Fedora CoreOS Config Transpiler, is a tool that produces a JSON Ignition file from the YAML FCC file. Using this configuration file, an FCOS machine can be told to create users, create filesystems, set up the network, install systemd units, and more.
+
+Download the latest version of `fcct` and the detached signature from the https://github.com/coreos/fcct/releases[releases page].
```

Comment:
Instead of downloading and running binaries from Github, I'd prefer to show the containerized flow here.

Can you please adapt this section to follow https://github.com/coreos/fcct/blob/master/docs/getting-started.md#container?

Reply:
Will do!